build/use_ccache.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
Running this as a script won't do anything, because it will only set the env vars in the subshell. So this doesn't need the `#!` line, and the instructions should make it clear that users need to say `source build/use_ccache.sh` or `. build/use_ccache.sh` in the terminal before running `cmake`.